spec/features/projects/project_status_administration_spec.rb

```diff
@@ -43,7 +43,7 @@
   let(:global_permissions) { [:add_project] }
   let(:project_permissions) { [:edit_project] }
   let!(:project_role) do
-    create(:role, permissions: project_permissions).tap do |r|
+    create(:project_role, permissions: project_permissions).tap do |r|
```

Comment:
Unrelated but can be bundled:
```suggestion
    create(:project_role, permissions: project_permissions) do |r|
```